adhocteam/Head-Start-TTADP
# Duration supports half hours

**Description of change**
Duration field now supports half steps

**How to test**
1) Pull down
2) Create a new report, enter half an hour for duration (*.5) and click `Continue`. Note no error
3) Enter a quarter of an hour for duration (*.25) and click `Continue`. Note you now see an error

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/307

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
